resdk/scripts.py

```diff
@@ -350,3 +352,122 @@ def upload_reads():
+        'pseudogene': 'pseudo',
+        'pseudo': 'pseudo',
+        'asRNA': 'asRNA',
+        'antisense': 'asRNA',
+        'protein_coding': 'protein-coding',
+        'protein-coding': 'protein-coding',
+        'IG_V_gene': 'protein-coding',
+        'IG_LV_gene': 'protein-coding',
+        'TR_C_gene': 'protein-coding',
+        'TR_V_gene': 'protein-coding',
+        'TR_J_gene': 'protein-coding',
+        'IG_J_gene': 'protein-coding',
+        'TR_D_gene': 'protein-coding',
+        'IG_C_gene': 'protein-coding',
+        'IG_D_gene': 'protein-coding',
+        'miRNA': 'ncRNA',
+        'lincRNA': 'ncRNA',
+        'processed_transcript': 'ncRNA',
+        'sense_intronic': 'ncRNA',
+        'sense_overlapping': 'ncRNA',
+        'bidirectional_promoter_lncRNA': 'ncRNA',
+        'ribozyme': 'ncRNA',
+        'Mt_tRNA': 'ncRNA',
+        'Mt_rRNA': 'ncRNA',
+        'misc_RNA': 'ncRNA',
+        'macro_lncRNA': 'ncRNA',
+        '3prime_overlapping_ncRNA': 'ncRNA',
+        'sRNA': 'ncRNA',
+        'snRNA': 'snRNA',
+        'scaRNA': 'snoRNA',
+        'snoRNA': 'snoRNA',
+        'rRNA': 'rRNA',
+        'ncRNA': 'ncRNA',
+        'tRNA': 'tRNA',
+        'other': 'other',
+        'unknown': 'unknown'
+    }
+
+    errors = 0
+    updated = 0
+    with zipfile.ZipFile(args.features) as archive:
+        for entry in archive.infolist():
+            if not entry.filename.endswith('.tab'):
+                continue
+
+            if args.verbose:
+                print('Importing features from "{}"...'.format(entry.filename))
+
+            reader = csv.DictReader(archive.open(entry), delimiter=str('\t'))
+            for row in reader:
+                aliases = row['Aliases'].strip()
+                if not aliases or aliases == '-':
+                    aliases = []
+                else:
+                    aliases = aliases.split(',')
+
+                try:
+                    resolwe.feature.post({
+                        'source': row['Source'],
+                        'feature_id': row['ID'],
+                        'species': row['Species'],
+                        'type': row['Type'],
+                        'sub_type': subtype_map.get(row['Gene type'], 'other'),
+                        'name': row['Name'],
+                        'full_name': row['Full name'],
+                        'description': row['Description'],
+                        'aliases': aliases
+                    })
+                    updated += 1
+                except slumber.exceptions.HttpClientError as error:
+                    print("WARNING: Failed to update feature '{}:{}':\n{}\n{}".format(
+                        row['Source'],
+                        row['ID'],
+                        row,
+                        error.response.content  # pylint: disable=no-member
+                    ))
+                    errors += 1
+
+    if errors:
+        print("Encountered {} errors during import.".format(errors))
```

Comment:
We use [BraceMessage](https://github.com/genialis/resolwe/blob/master/resolwe/utils.py#L4) in resolwe and resolwe-bio to avoid the formatting issue. See a usage example [here](https://github.com/genialis/resolwe/blob/3e4bc3211ccc03eca79647ca988bd0afb91b3b25/resolwe/flow/managers/base.py#L22). You can add an exception for the invalid constant name. Can you tell pylint that logger is an OK constant?